api/src/main/java/com/trainlab/controller/FrontendDataController.java

```diff
@@ -30,8 +25,8 @@
 @Tag(name = "FrontendDataController", description = "Frontend data")
 @Slf4j
 public class FrontendDataController {
```

Comment:
Создать интерфейс FrontendDataController и перенести в него описание

```suggestion
public class FrontendDataControllerImpl implements FrontendDataController{
```